infrastructure/main.tf

```diff
@@ -28,14 +34,21 @@ module "pcq-db-flexible" {
   product       = var.product
   component     = var.component
   business_area = "cft"
-  name          = "${var.product}-${var.component}-flexible"
+  name          = local.db_server_name
 
   common_tags = var.common_tags
 
   pgsql_admin_username = "pcquser"
   pgsql_version        = "15"
   auto_grow_enabled    = true
 
+  action_group_name           = join("-", [var.db_monitor_action_group_name, local.db_server_name, var.env])
+  email_address_key           = var.db_alert_email_address_key
+  email_address_key_vault_id  = data.azurerm_key_vault.key_vault.id
+  cpu_threshold               = var.cpu_threshold
```

Comment:
if you want to use default values of 80% then no need to define these here
cpu_threshold, memory_threshold and storage_threshold can be removed from here